kolibri/plugins/coach/assets/src/views/CoachSideNavEntry.vue

```diff
@@ -2,8 +2,8 @@
 
   <CoreMenuOption
     :label="coreString('coachLabel')"
-    :link="url"
```

Comment:
Just to make sure I am understanding correctly - the top level option now no longer links, and instead reveals the top level nav items to click on?

Reply:
Correct, unless there are not other nav items (i.e. profile page)